app/helpers/prefix-http.js

```diff
@@ -1,6 +1,9 @@
 import Ember from 'ember'
 
 export function prefixHttp([ url ]) {
+  if (url === undefined) {
```

Comment:
I'd check for `url == null` this catches `undefined` and `null` (only!). This is probably one of the only legitimate uses for `==` :)